.rubocop.yml

```diff
@@ -12,3 +12,27 @@ Style/RegexpLiteral:
     - percent_r
     - mixed
 
+  AllowInnerSlashes: true
+
+Style/Documentation:
+  Enabled: false
+
+Metrics/MethodLength:
+  Max: 17
```

Comment:
Honestly I'd just disable methodlength, classlength, etc. They're crappy complexity metrics. Things like ABC size and perceived complexity are much more meaningful.

These length ones just encourage you to sacrificed readability in favour of fewer lines in my experience.